mmv1/products/looker/Instance.yaml

```diff
@@ -0,0 +1,368 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Instance'
+base_url: projects/{{project}}/locations/{{region}}/instances
+create_url: projects/{{project}}/locations/{{region}}/instances?instanceId={{name}}
+update_verb: :PATCH
+update_mask: true
+description: |
+  A Google Cloud Looker instance.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Create a Looker (Google Cloud core) instance': 'https://cloud.google.com/looker/docs/looker-core-instance-create'
+    'Configure a Looker (Google Cloud core) instance': 'https://cloud.google.com/looker/docs/looker-core-instance-setup'
+  api: 'https://cloud.google.com/looker/docs/reference/rest/v1/projects.locations.instances'
+timeouts: !ruby/object:Api::Timeouts
```

Comment:
Unless you'd strongly prefer the shorter timeouts, I'd lean towards 90 across-the-board!

Reply:
Done.